src/Selenium2Library/locators/elementfinder.py

```diff
@@ -4,6 +4,25 @@
 from Selenium2Library.utils import escape_xpath_value, events
 
 
+class ParserLocator(object):
+
+    @classmethod
+    def parse_locator(cls, locator):
```

Comment:
- What benefits does making this a class method bring?
- Since this is a method in a locator parser class, it could be named simply `parse`. Namespaces are a honking good idea and all that.